mautic/mautic
# Ensure test mail fixture is a valid mail object with custom HTML

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13106 



#### Description:

This PR addresses fixture data that is no longer in line with the code and flows in Mautic

The `LoadClickData` fixture creates an email without `CustomHtml` or `template, which results in the error in  #13106

this PR addresses this



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. execute the `send email` action and select the `Email test` mail
3. verify there were no errors and the mail is sent / visible in mailhog
